standard/standard.md

```diff
@@ -0,0 +1,80 @@
+# Accelerator Lattace Standard
+
+## Overview
+
+The Accelerator Lattace Standard (ALS) defines a standard for the sharing of lattice information to describe
+particle accelerators and storage rings. ALS aims to promote:
+
+ - portability between various applications and differing algorithms
+ - a unified open-access description for scientific data (publishing and archiving)
+ - a unified description for post-processing, visualization and analysis.
+
+ALS is able to describe the connections between various things
+from the connection of injection and extraction lines connected to a storage ring to the interaction region
+of colliding beam storage rings where particles are moving through magnets in opposite directions. An ALS
+based lattice is able to
+hold all the information about an entire machine complex from beam creation to dump lines enabling a 
+single lattice to be used as the basis of start-to-end simulations.
+
+ALS is built to be easily customizable so that custom information may be interted by a program into a lattice.
+This custom information is generally not usable by other programs but can be useful when a program accesses
+lattice files that it generated. 
+
+
+## What ALS Is
+
+ALS is a schema that defines things like the names of various lattice element types, how to organize lattice
+elements into lines which beams of particles or X-rays can move through, etc. 
+
+## What ALS Is Not
+
+ALS does not define any particular grammer to implemement the ALS schema. Rather, there are associated
+language specific standards that define grammers for YAML, JSON, Python, etc. Along with these
+associated standards, there are packages that implement translation between lattice files and a representational
+internal format defined by the package.
+
+ALS does not define how particles are to be tracked through a lattice. ALS is for describing machines and
+not for defining how to simulate particle motion. 
+
+## Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+All `keywords` in this standard are case-sensitive.
+
+## Lattice Elements
+
+The basic building block used to describe an accelerator is the lattice \vn{element}. Typically,
+a lattice element is something physical like a bending magnet or an electrostatic
+quadrupole, or a diffracting crystal. A lattice element may define a region in space 
+distinguished by the presense of (possibly time-varying) electromagnetic fields,
+materials, apertures and other possible engineered structures. However, lattice elements
+are not restricted to being something physical and may, for example, just mark a particular point in space
+(EG: **Marker** elements), or may designate where beam lines intersect (**Fork** elements).
+By convention, element names in ALS will be upper camel case.
+
```

Comment:
The order of the elements must correspond to the physical sequence.

Reply:
I put in some words on this.